packages/analytics-browser/src/config.ts

```diff
@@ -213,10 +213,9 @@ export const useBrowserConfig = async (
   amplitudeInstance.previousSessionUserId = previousCookies?.userId ?? legacyCookies.userId;
 
   const trackingOptions = {
-    ipAddress: true,
-    language: true,
-    platform: true,
-    ...options.trackingOptions,
+    ipAddress: options.trackingOptions?.ipAddress ?? true,
+    language: options.trackingOptions?.ipAddress ?? true,
+    platform: options.trackingOptions?.ipAddress ?? true,
```

Comment:
These don't match

Reply:
Just saw it after creating this PR. Fixed in the later commit.